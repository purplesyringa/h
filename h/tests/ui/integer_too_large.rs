fn main() {
    let meow = h::map!(
        for u8;
        256 => (),
    );

    let mrrrp = h::map!(
        for i8;
        -129 => (),
    );

    let absurdly_large = h::map! {
        for _;
        1_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_u128 => (),
    };
}
