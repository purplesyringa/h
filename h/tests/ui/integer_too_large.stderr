error: too large integer (doesn't fit in `u8`)
 --> tests/ui/integer_too_large.rs:4:9
  |
4 |         256 => (),
  |         ^^^

error: too large integer (doesn't fit in `i8`)
 --> tests/ui/integer_too_large.rs:9:9
  |
9 |         -129 => (),
  |         ^^^^

error: too large integer
  --> tests/ui/integer_too_large.rs:14:9
   |
14 |         1_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_u128 => (),
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
